plugins/ralph-speckit/.specify/scripts/bash/check-prerequisites.sh

```
@@ -0,0 +1,166 @@
+            echo "ERROR: Unknown option '$arg'. Use --help for usage information." >&2
+            exit 1
+            ;;
+    esac
+done
+
+# Source common functions
+SCRIPT_DIR="$(CDPATH="" cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
+
+# Get feature paths and validate branch
+eval $(get_feature_paths)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)